R/annotation.R

```diff
@@ -90,14 +92,30 @@ makeOrgGatomAnnotation <- function(org.db,
     pathways <- getMetabolicPathways(org.gatom.anno$genes$gene, keggOrgCode,
                                      includeReactome=(org.gatom.anno$baseId == "ENTREZID"))
 
-    # keeping only pathways with at least half of enzymatic genes
-    pathwaysEnzymeRatio <-
-      lengths(lapply(pathways, intersect, y=unique(org.gatom.anno$gene2enzyme$gene)))/
-      lengths(pathways)
-    pathways <- pathways[pathwaysEnzymeRatio >= 0.5]
+    # keeping only metabolic pathways (by Reactome tree and metabolic genes enrichment):
+    ids.reactome.relation <- read.table("https://reactome.org/download/current/ReactomePathwaysRelation.txt")
+    colnames(ids.reactome.relation) <- c("id", "child")
 
-    org.gatom.anno$pathways <- pathways
+    id.Metabolism <- switch(
```

Comment:
Pathway name "Metabolism" should be used instead of paritcular pathway IDs, so it's compatible with any organism